operations/commit_queue.go

```diff
@@ -398,21 +411,24 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 		return errors.New("module patch aborted")
 	}
 
+	owner, repo, err := thirdparty.ParseGitUrl(module.Repo)
+	if err != nil {
+		return errors.Wrapf(err, "can't get owner/repo from '%s'", module.Repo)
+	}
+
 	patch, err := rc.GetPatch(p.patchID)
 	if err != nil {
 		return errors.Wrapf(err, "can't get patch '%s'", p.patchID)
 	}
 
-	if err = isValidCommitsFormat(p.commits); err != nil {
-		return err
+	commitMessages, err := gitCommitMessages(module.Branch, p.ref, p.commits)
+	if err != nil {
+		return errors.Wrap(err, "can't get module commit messages")
 	}
-
-	message := ""
-	if patch.Description == "" {
-		message, err = gitCommitMessages(module.Branch, p.ref, p.commits)
-		if err != nil {
-			return errors.Wrap(err, "can't get module commit messages")
-		}
+	commits := fmt.Sprintf(commitFmtString, commitMessages, owner, repo, module.Branch)
```

Comment:
Can you get move lines 414-418 & 424-428 into the if statement at 430? Doesn't look like they're used elsewhere.

Reply:
my intention was: 
If the patch already contains commits we append to the existing description with this module's commits. This is the message on 429.
If the original patch has no commits (so it's a module-only item) the module commits' description will replace the original description. This is the message on 431.

Reply:
But what I mean is: all of these things only need to happen IF the patch description has the suffix noCommits.

Reply:
Oh I see what you're saying. Can you add a comment to explain this logic?